python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json

```
@@ -0,0 +1,331 @@
+               ".",
+               ".",
+               ".",
+               "loop1",
+               ".",
+               "squashfs"
+            ],
+            [
+               "...",
+               "/snap/amazon-ssm-agent/1480",
+               ".",
+               ".",
+               ".",
+               "loop3",
+               ".",
+               "."
+            ]
+         ],
+         "region":"{region}",
+         "title":"Disk used"
+      }
+   },
+   {
+      "type":"metric",
+      "x":12,
+      "y":18,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "mem_used_percent",
+               "InstanceId",
+               "{instance_id}"
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"Memory used"
+      }
+   },
+   {
+      "height": 6,
+      "width": 12,
+      "y": 0,
+      "x": 0,
+      "type": "metric",
+      "properties": {
+          "metrics": [
+              [ { "expression": "SEARCH('{{cluster_name}-ray-prometheus,instance} ray_node_cpu_count', 'Maximum', 300)", "id": "e1", "period": 300 } ],
+              [ { "expression": "SUM(e1)", "label": "Expression2", "id": "e2" } ]
+          ],
+          "view": "timeSeries",
+          "stacked": false,
+          "region": "{region}",
+          "stat": "Maximum",
+          "period": 300,
+          "start": "-PT2H",
+          "end": "P0D",
+          "title": "Cluster CPU Count"
+      }
+   },
+   {
+      "height": 6,
+      "width": 12,
+      "y": 0,
+      "x": 12,
+      "type": "metric",
+      "properties": {
+          "metrics": [
+              [ { "expression": "SEARCH('{{cluster_name}-ray-prometheus,instance} object_store_available_memory', 'Average', 300)", "id": "e3", "period": 300 } ],
+              [ { "expression": "SUM(e3)", "label": "Expression1", "id": "e1" } ]
+          ],
+          "view": "timeSeries",
+          "stacked": false,
+          "region": "{region}",
+          "stat": "Maximum",
+          "period": 300,
+          "start": "-PT2H",
+          "end": "P0D",
+          "title": "Cluster Object Store Available Mem"
+      }
+   },
+   {
+      "height": 6,
+      "width": 12,
+      "y": 6,
+      "x": 12,
+      "type": "metric",
+      "properties": {
+          "metrics": [
+              [ { "expression": "SUM(e2)", "label": "Expression1", "id": "e1" } ],
+              [ { "expression": "SEARCH('{{cluster_name}-ray-prometheus,instance} MetricName=\"ray_node_mem_available\"', 'Average', 300)", "id": "e2", "period": 300, "visible": false } ]
+          ],
+          "view": "timeSeries",
+          "stacked": false,
+          "region": "{region}",
+          "stat": "Maximum",
+          "period": 300,
+          "start": "-PT2H",
+          "end": "P0D",
+          "title": "Ray Cluster Available Mem"
```

```suggestion
          "title": "Cluster Available Mem"
```
Since the user is assigning the dashboard name for their Ray Cluster, I think we can save a bit of space by omitting `Ray` from their titles.